specification/iothub/resource-manager/Microsoft.Devices/2017-07-01/iothub.json

```diff
@@ -1013,9 +1013,452 @@
         },
         "deprecated": false
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/certificates": {
+      "get": {
+        "tags": [ "GET" ],
+        "summary": "Get the certificate list.",
+        "description": "Get the certificate list.",
+        "operationId": "IotHubResource_ListCertificates",
```

Comment:
For better user experience this operation Id should be `Certificates_ListByIoTHub`, this will ensure all the operations on certificates grouped under the `Certificates` class/interface.